setup.py

```diff
@@ -2,5 +2,6 @@
 import setuptools
 
 setuptools.setup(
+    name='mock',
```

Comment:
Could you explain why this is needed / desirable? I don't like the duplication of metadata that this creates.

Reply:
This keyword is required by Buildroot's script [scanpypi](https://git.busybox.net/buildroot/tree/utils/scanpypi) in order to automatically add Python package to Buildroot's tree. At the point of invocation setup metadata won't be expanded though pbr, so 'name' is missing.

So basically my PR should remove 'name' from setup.cfg and add it to setup.py.

Reply:
Well pip manages to figure it out just fine - I think you've got a bug in scanpypi.